app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,42 @@
+<%= render partial: 'partials/navigation_menu', locals: {visited: 'recipes'} %>
+<main>
+  <h2><%= @recipe.name %></h2>
+  <div>
+    <p><span>Preparation Time:</span> <%= @recipe.preparation_time %></p>
+    <%= render partial: 'switch_button', locals: {public: @recipe.public} %>
+  </div>
```

Comment:
- [ ] The preparation time I am seeing on the page is not one that I have put in while creating the recipe. I think this arises from the creation of the recipe. If you see the previous comments, you will notice that the `:preparation_time` has been assigned to two places with different types. Please fix that issue so that the recipe page will show the proper preparation time 😃 👍 
![image](https://user-images.githubusercontent.com/90222110/218268766-2b32107e-e274-4580-a20d-b87355452ab0.png)